lm75bd/lm75bd.c

```diff
@@ -26,9 +26,41 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
   return ERR_CODE_SUCCESS;
 }
 
+
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
   /* Implement this driver function */
-  
+  //sets the temp register
+  error_code_t errCode;
+
+  if ( temp == NULL){
+    return ERR_CODE_INVALID_ARG;
+  }
+  uint8_t buff[1] = {0};
```

Comment:
The array notation is unnecessary. Since uint8_t is already 1 byte, you can just pass in the address to buff in i2cSendTo call. So it'll look like this: i2cSendTo(devAddr, &buff, 1).